guillotina_gcloudstorage/storage.py

```diff
@@ -366,8 +365,8 @@ def _get_access_token(self):
         return self._get_access_token()
 
     def _get_or_create_bucket(self, bucket_name):
-        client = google.cloud.storage.Client(
-            project=self._project, credentials=self._credentials)
+        client = google.cloud.storage.Client.from_service_account_json(
```

Comment:
should it be something like `if not self._project:`. Or is it that latest google cloud no longer supports the old api?

Should we pin a range of google api versions it's compatible with?

Reply:
The last google does not support the old api, but on the json credentials there is the project, so its duplicating information.

Reply:
Should we add a version range to setup.py?

Reply:
It should work on older versions